From 485b2d605e263159494621264a734fe1b4376f8e Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Mon, 6 Mar 2017 18:39:30 +0200
Subject: [PATCH 1437/2241] arm: defconfig: enable tcrypt testing module

tcrypt is a simple testing module which allows testing
Kernel crypto (data integrity and benchmarks).

Change-Id: I3c3ceb45738c50d1314458c2fb03203cdaba7f77
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37249
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm/configs/mvebu_v7_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/mvebu_v7_lsp_defconfig b/arch/arm/configs/mvebu_v7_lsp_defconfig
index b78134b..1e6c9e1 100644
--- a/arch/arm/configs/mvebu_v7_lsp_defconfig
+++ b/arch/arm/configs/mvebu_v7_lsp_defconfig
@@ -183,4 +183,5 @@ CONFIG_MAGIC_SYSRQ=y
 CONFIG_TIMER_STATS=y
 # CONFIG_DEBUG_BUGVERBOSE is not set
 CONFIG_DEBUG_USER=y
+CONFIG_CRYPTO_TEST=m
 CONFIG_CRYPTO_DEV_MARVELL_CESA=y
-- 
2.7.4

